src/hrtf/mysofa.h

```diff
@@ -29,6 +29,13 @@ struct MYSOFA_ARRAY {
 	struct MYSOFA_ATTRIBUTE *attributes;
 };
 
+/** additional variable */
+struct MYSOFA_VARIABLE {
+	struct MYSOFA_VARIABLE *next;
+	char *name;
+	struct MYSOFA_ARRAY *value;
```

Comment:
why an array not a string?

Reply:
Because that way it's possible to store sequences of float values without having to parse them from a string. Strings could already be stored in the attributes before, but that's fairly inconvenient in my use case.

Reply:
Attributes only allow string values, variables can be arrays.